clients/go/admin/deviceflow/config.go

```diff
@@ -0,0 +1,10 @@
+package deviceflow
+
+import "github.com/flyteorg/flytestdlib/config"
+
+// Config defines settings used for Device orchestration flow.
+type Config struct {
+	TokenRefreshGracePeriod config.Duration `json:"refreshTime" pflag:",grace period from the token expiry after which it would refresh the token."`
+	Timeout                 config.Duration `json:"deviceFlowTimeout" pflag:",amount of time the device flow should complete or else it will be cancelled."`
```

Comment:
```suggestion
	Timeout                 config.Duration `json:"timeout" pflag:",amount of time the device flow should complete or else it will be cancelled."`
```